endless-mount-dirs.service

```diff
@@ -25,9 +25,10 @@ RequiresMountsFor=/var
 # Only run if the directories don't exist
 ConditionPathIsDirectory=|!/var/endless
 ConditionPathIsDirectory=|!/var/endless-extra
+ConditionPathIsDirectory=|!/var/workdir
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/bin/mkdir -p /var/endless /var/endless-extra
+ExecStart=/bin/mkdir -p /var/endless /var/endless-extra /var/workdir
 ExecStart=-/bin/chown app-manager:app-manager /var/endless /var/endless-extra
```

Comment:
Ow, wait, are you referring to manually creating the directory, or to changing it's ownership to "app-manager". Regarding the later, I've asked myself the same question, but since I was able to write in the overlay (/endless) as the app-manager user, I don't think it is necessary.